continuous_integration/environment-3.7.yaml

```diff
@@ -5,23 +5,25 @@ channels:
 dependencies:
   # required dependencies
   - python=3.7
-  - numpy=1.17
-  - pandas=1.1
+  - numpy=1.15
+  - pandas=0.25
   # test dependencies
   - pytest
   - pytest-cov
   - pytest-rerunfailures
   - pytest-xdist
   - moto
   - flask
-  - fastparquet
+  # Issue with conda package for NumPy 1.15
+  # See fastparquet-444
+  # - fastparquet
   - h5py
   - pytables
   - zarr
   - tiledb-py
   - fsspec
   - sqlalchemy
-  - pyarrow=1.0
+  - pyarrow=0.14
```

Comment:
Also this one (cc @jorisvandenbossche)

Reply:
- environment-3.6 used to pin an obsolete version of s3fs; shouldn't the same be moved to 3.7?
- we don't have a <3.8 environment were pickle5 isn't installed anymore; should we care?